catalogue/webapp/components/ArchiveTree/ArchiveTree.tsx

```diff
@@ -184,9 +187,16 @@ const StyledLink = styled.a<StyledLinkProps>`
       : 0};
   padding-right: ${props => `${props.theme.spacingUnit * 2}px`};
   text-decoration: none;
+
+  &:focus-visible,
   &:focus {
```

Comment:
Maybe silly question, this is interesting! Is `&:focus` still needed here if we just want it to be on `&:focus-visible`?

Reply:
Oh that does make sense, thank you! I hadn't gotten to that point. 🙌